website/src/client/components/Import/ImportFilesManager.tsx

```diff
@@ -136,7 +138,7 @@ export default class ImportFilesManager extends React.PureComponent<Props, State
         folderMappings[item.name] = getUniquePath(allPaths, item.name);
       }
     });
-
+    // @ts-ignore
```

Comment:
What's the error you get for this one?

Reply:
<img width="844" alt="Screen Shot 2022-04-15 at 4 03 02 PM" src="https://user-images.githubusercontent.com/5482800/163651172-3c62143a-eba2-422c-9cea-83f8b6045e84.png">